master/usage/configuration/bgp.md

```diff
@@ -96,21 +96,54 @@ to ensure continuity of service.
 
 #### Example
 
-To turn off the full BGP node-to-node mesh, or to modify the global AS number,
-run the following command on any node:
-
-```
-# Get the current bgpconfig settings
-$ calicoctl get bgpconfig -o yaml > bgp.yaml
-
-# Modify nodeToNodeMeshEnabled to true or false
-# -or-
-# modify the asNumber to appropropriate integer
-$ vim bgp.yaml
-
-# Replace the current bgpconfig settings
-$ calicoctl replace -f bgp.yaml
-```
+**Prerequisite**: `calicoctl` [installed]({{site.baseurl}}/{{page.version}}/usage/calicoctl/install) and [configured]({{site.baseurl}}/{{page.version}}/usage/calicoctl/configure/).
+
+1. Issue the following command to determine if you have a `default` BGP configuration
+   resource.
+   
+    ```
+    calicoctl get bgpconfig default
+    ```
+
+1. If the resource _does not_ exist, skip to step 3. Otherwise, use the following 
```

Comment:
does not? Shouldn't it be "skip to step 3 if the resource already exists" ?

Reply:
Ooooops yeah 😄